lib/fetch-package-metadata.js

```diff
@@ -32,6 +33,38 @@ const CACHE = new LRUCache({
   length: (p) => p._contentLength
 })
 
+const FetchConfig = figgyPudding({
```

Comment:
This is unnecessary if we go the "no-figgy-pudding" approach.

Reply:
I think this module disappears anyway when we migrate to arborist.